app/views/user_emails/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= render(PageComponent.new(subject: @user, sidebar: false)) do |component| %>
+  <% component.with(:header) do %>
+    <%= render(Page::HeaderComponent.new(
+          title: @user&.name,
+          subtitle: @user.email,
+          back_link: BackLinkComponent.new(title: "Emails", link: user_emails_path(current_user)),
+          sidebar: true, buttons: [
+            layout_button(link: edit_user_path(@user), icon: "fad fa-pencil", title: "Edit user"),
+            layout_button(link: user_impersonation_path(@user), method: :put, icon: "fad fa-mask", title: "Masquerade as user", admin: true)
+          ])) %>
+  <% end %>
+  <% component.with(:body) do %>
+    <%= render CardComponent.new(classes: "trix-email-preview") do %>
+      <%= render ListGroupComponent.new(classes: "list-group-messages list-group-flush list-group-bordered") do %>
```

Comment:
This is one way of doing it - though `ListGroupComponent` has some args to make this easier. See [the class initializer](https://github.com/gitcoinco/code_fund_ads/blob/master/app/components/list_group_component.rb#L2)

`<%= render ListGroupComponent.new(bordered: true, flush: true) do %>`

I would suggest just adding messages as another named arg to the component so the component would then look like:

```ruby
class ListGroupComponent < ApplicationComponent
  def initialize(bordered: false, flush: false, reflow: false, messages: false, classes: nil)
    @bordered = bordered
    @flush = flush
    @reflow = reflow
    @messages = messages
    @class_names = classes
  end

  private

  attr_reader :class_names, :bordered, :flush, :reflow, :messages

  def classes
    classes = ["list-group"]
    classes << "list-group-bordered" if bordered
    classes << "list-group-flush" if flush
    classes << "list-group-reflow" if reflow
    classes << "list-group-messages" if messages
    classes << class_names if class_names
    classes.compact
  end

  def render?
    content.present?
  end
end
```

Since we are using the messages class in more than one place I would recommend doing it this way, which would make your render `<%= render ListGroupComponent.new(bordered: true, flush: true, messages: true) do %>` but you could just not add messages and do `<%= render ListGroupComponent.new(bordered: true, flush: true, classes: "list-group-messages") do %>`. Reducing the use of the `classes` escape hatch will make updating styles easier though since there is a single source of truth.

Reply:
You realize how much complexity is being added to a simple div right?

Reply:
Also if you add `messages`, this would be a great test to add to `test/components/list_group_component_test.rb`

```ruby
  test "messages list group" do
    assert_equal(
      %(<ul class="list-group list-group-messages"> List Group </ul>),
      render_inline(ListGroupComponent.new(messages: true)) { "List Group" }.to_html.squish
    )
  end
```